### PR TITLE
Docstring Changes

### PR DESCRIPTION
**Pull request by jm786**
### Summary of PR:
Corrected docstrings (grammar, meaning, etc.) and docstring examples (rounded down to 2 decimal places where appropriate) in pycgmStatic.py

### What issues does this PR close:
Grammar and Explanations in documentation.

### What files were changed and what changes were made?
pycgmStatic.py: Fixed docstrings and rounded example values to 2 decimal places.

### Does your PR (check all that applies):
 

- [ ] Add documentation
- [x]  Change/Remove documentation
- [ ]  Add tests
- [ ]  Change/Remove tests
- [ ]  Add code
- [ ]  Change/Remove code
- [x]  Fix formatting

### Are there any errors/relevant logs in your code?
None

### How has this been tested?
Used pytest and passed both repo checks.

### CODE REVIEW

File: pyCGM_Single/pycgmStatic.py [hunk 1]
```diff
@@ -428,17 +430,17 @@ def headoffCalc(axisP, axisD):
...
-    Takes in anatomical uncorrect axis and anatomical correct axis.
-    Correct axis depends on foot flat options.
+    Takes in anatomically incorrect axis and anatomically correct axis.
+    Corrects the axis depending on flat-footedness.
 
-    Calculates the offset angle between that two axis.
+    Calculates the offset angle between those two axes.
 
-    It is rotated from uncorrect axis in YXZ order.
+    It is rotated from incorrect axis in YXZ order.
```
Comment (cadop) [inline pyCGM_Single/pycgmStatic.py:438]: uncorrected

File: pyCGM_Single/pycgmStatic.py [hunk 2]
```diff
@@ -428,17 +430,17 @@ def headoffCalc(axisP, axisD):
 def staticCalculation(frame,ankle_JC,knee_JC,flat_foot,vsk=None):
     """Calculate the Static angle function
 
-    Takes in anatomical uncorrect axis and anatomical correct axis.
-    Correct axis depends on foot flat options.
+    Takes in anatomically incorrect axis and anatomically correct axis.
```
Comment (cadop) [inline pyCGM_Single/pycgmStatic.py:433]: uncorrected

File: pyCGM_Single/pycgmStatic.py [hunk 3]
```diff
@@ -451,47 +453,47 @@ def staticCalculation(frame,ankle_JC,knee_JC,flat_foot,vsk=None):
...
     angle : list
-        Returns the offset angle represented by a 2x3x3 array.
+        Returns the offset angle represented by a 2x3x3 list.
         The array contains the right flexion, abduction, rotation angles (1x3x3)
         followed by the left flexion, abduction, rotation angles (1x3x3).
 
     Modifies
     --------
-    The correct axis changes following to the foot flat option.
+    The correct axis changes depending on flat-footedness.
```
Comment (cadop) [inline pyCGM_Single/pycgmStatic.py:462]: `flat foot` is correct.

File: pyCGM_Single/pycgmStatic.py [hunk 4]
```diff
@@ -1192,19 +1194,21 @@ def ankleJointCenter(frame,knee_JC,delta,vsk=None):
...
+    In the case of the foot joint center, we've already made 2 kinds of axes
+    for the static offset angle and then, we call this static offset angle as an
+    input of this function for thedynamic trial.
 
     Special Cases:
 
-    (anatomical uncorrect foot axis)
-    if foot flat is checked, make the reference markers instead of HEE marker which height is as same as TOE marker's height.
-    elif foot flat is not checked, use the HEE marker for making Z axis.
+    (anatomically incorrect foot axis)
```
Comment (cadop) [inline pyCGM_Single/pycgmStatic.py:1209]: uncorrected

File: pyCGM_Single/pycgmStatic.py [hunk 5]
```diff
@@ -1192,19 +1194,21 @@ def ankleJointCenter(frame,knee_JC,delta,vsk=None):
...
+    If flat-footed, make the reference markers instead of HEE marker whose height
```
Comment (cadop) [inline pyCGM_Single/pycgmStatic.py:1210]: flat foot

File: pyCGM_Single/pycgmStatic.py [hunk 6]
```diff
@@ -1225,60 +1229,60 @@ def footJointCenter(frame,static_info,ankle_JC,knee_JC,delta):
 
     Returns
     -------
-    R, L, foot_axis : array
-        Returns an array that contains the foot axis center in a 1x3 array of xyz values,
-        which is then followed by a 2x3x3 array composed of the foot axis center x, y, and z
-        axis components. The xyz axis components are 2x3 arrays consisting of the axis center
+    R, L, foot_axis : list
+        Returns a list that contain the toe axis' (right and left) origin in 1x3 arrays
```
Comment (cadop) [inline pyCGM_Single/pycgmStatic.py:1233]: foot, not toe

File: pyCGM_Single/pycgmStatic.py [hunk 7]
```diff
@@ -1540,26 +1543,26 @@ def headJC(frame):
     return [head_axis,origin]
 
 def uncorrect_footaxis(frame,ankle_JC):
-    """Calculate the anatomical uncorrect foot joint center and axis function.
+    """Calculate the anatomically incorrect foot joint center and axis function.
```
Comment (cadop) [inline pyCGM_Single/pycgmStatic.py:1546]: uncorrected

**Review by cadop — CHANGES_REQUESTED** [review]

File: pyCGM_Single/pycgmStatic.py [hunk 8]
```diff
@@ -867,37 +869,37 @@ def kneeJointCenter(frame,hip_JC,delta,vsk=None):
...
     >>> kneeJointCenter(frame,hip_JC,delta,vsk) #doctest: +NORMALIZE_WHITESPACE
-    [array([413.21007973, 266.22558784, 464.66088466]),
-    array([143.55478579, 279.90370346, 524.78408753]),
-    array([[[414.20806312, 266.22558785, 464.59740907],
-    [413.14660414, 266.22558786, 463.66290127],
-    [413.21007973, 267.22558784, 464.66088468]],
-    [[143.65611281, 280.88685896, 524.63197541],
-    [142.56434499, 280.01777942, 524.86163553],
-    [143.64837987, 280.0465038 , 525.76940383]]])]
+    [array([364.24453179, 292.34485083, 515.31187596]),
```
Comment (cadop) [inline pyCGM_Single/pycgmStatic.py:895]: ...

File: pyCGM_Single/pycgmStatic.py [hunk 9]
```diff
@@ -1023,40 +1025,40 @@ def ankleJointCenter(frame,knee_JC,delta,vsk=None):
...
     >>> ankleJointCenter(frame,knee_JC,delta,vsk) #doctest: +NORMALIZE_WHITESPACE
-    [array([393.76181609, 247.67829633,  87.73775041]),
-    array([ 98.74901939, 219.46930221,  80.63068161]),
-    [[array([394.48171575, 248.37201349,  87.715368  ]),
-    array([393.07114385, 248.39110006,  87.61575574]),
-    array([393.69314056, 247.78157916,  88.73002876])],
-    [array([ 98.47494966, 220.42553804,  80.52821783]),
-    array([ 97.79246671, 219.20927276,  80.76255902]),
-    array([ 98.84848169, 219.60345781,  81.61663776])]]]
+    [array([393.76163633, 247.67765724,  87.73637664]),
```
Comment (cadop) [inline pyCGM_Single/pycgmStatic.py:1054]: ...

File: pyCGM_Single/pycgmStatic.py [hunk 10]
```diff
@@ -1225,60 +1229,60 @@ def footJointCenter(frame,static_info,ankle_JC,knee_JC,delta):
...
-    array([[[442.8881541 , 381.76460597,  43.64802096],
-            [441.89515447, 382.00308979,  42.66971773],
-            [442.44573691, 380.70886969,  42.81754643]],
-           [[ 39.50785213, 382.67891581,  42.75880631],
-            [ 38.49231839, 382.14765966,  41.93027863],
-            [ 39.75805858, 381.51956227,  41.98854914]]])]
+    [array([442.82, 381.62,  42.66]),
+    array([ 39.44, 382.45,  41.79]),
+    array([[[442.88903073, 381.7602408 ,  43.64770809],
+            [441.89356133, 381.9962754 ,  42.67132275],
```
Comment (cadop) [inline pyCGM_Single/pycgmStatic.py:1281]: ...

**Review by cadop — CHANGES_REQUESTED** [review]

**Comment by jm786** [comment]
I made the changes requested; all doctests are now rounded to two decimal places.

File: pyCGM_Single/pycgmStatic.py [hunk 11]
```diff
@@ -699,15 +701,15 @@ def hipJointCenter(frame,pel_origin,pel_x,pel_y,pel_z,vsk=None):
...
+    >>> vsk = {'MeanLegLength': 940.0, 'R_AsisToTrocanterMeasure': 72.51,
+    ...        'L_AsisToTrocanterMeasure': 72.51, 'InterAsisDistance': 215.91}
+    >>> pel_origin = [ 251.61, 391.74, 1032.89]
+    >>> pel_x = [251.74, 392.73, 1032.79]
+    >>> pel_y = [250.62, 391.87, 1032.87]
+    >>> pel_z = [251.60, 391.85, 1033.89]
     >>> hipJointCenter(frame,pel_origin,pel_x,pel_y,pel_z,vsk)
-    array([[182.57097799, 339.43231799, 935.52900136],
-           [308.38050352, 322.80342433, 937.98979092]])
+    array([[183.23960265, 338.80485163, 934.65192157],
```
Comment (cadop) [inline pyCGM_Single/pycgmStatic.py:711]: rounding

**Review by cadop — CHANGES_REQUESTED** [review]
One issue with rounding. 

test_pycgmStatic is not a docstring, its a unittest. Please undo.

**Review by cadop — APPROVED** [review]

**Pull request merged**